lib/js-yaml/dumper.js

```diff
@@ -194,14 +194,7 @@ function isPlainSafe(c) {
   // where nb-char ::= c-printable - b-char - c-byte-order-mark.
   return isPrintable(c) && c !== 0xFEFF
     // - c-flow-indicator
-    && c !== CHAR_COMMA
-    && c !== CHAR_LEFT_SQUARE_BRACKET
-    && c !== CHAR_RIGHT_SQUARE_BRACKET
-    && c !== CHAR_LEFT_CURLY_BRACKET
-    && c !== CHAR_RIGHT_CURLY_BRACKET
-    // - ":" - "#"
-    && c !== CHAR_COLON
-    && c !== CHAR_SHARP;
+    && c !== CHAR_COLON;
```

Comment:
It would be nice to cover this change with tests. For example, check dumping (and load back) of this strings:

`C#`, `C#123`, `C #123`, `C# 123`